cabot/cabotapp/models.py

```diff
@@ -237,17 +237,35 @@ def recent_snapshots(self):
             s['time'] = time.mktime(s['time'].timetuple())
         return snapshots
 
+    # order checks by: critical + failing, error + failing, warning + failing, passing, disabled
+    _CHECK_ORDER = "(CASE " \
+                   "WHEN calculated_status = '{failing}' AND importance = '{critical}' THEN 1 " \
+                   "WHEN calculated_status = '{failing}' AND importance = '{error}' THEN 2 " \
+                   "WHEN calculated_status = '{failing}' AND importance = '{warning}' THEN 3 " \
+                   "WHEN active = false THEN 99 " \
+                   "ELSE 4 " \
+                   "END)".format(failing=CALCULATED_FAILING_STATUS,
+                                 intermittent=CALCULATED_INTERMITTENT_STATUS,
+                                 passing=CALCULATED_PASSING_STATUS,
+                                 critical=CRITICAL_STATUS,
+                                 error=ERROR_STATUS,
+                                 warning=WARNING_STATUS)
```

Comment:
I don't love the raw SQL but can't really think of a better way of doing it so seems legit